app/controllers/PensionSchemeMembersController.scala

```diff
@@ -43,36 +43,53 @@ import pages.PensionSchemeMembersPage
 import play.api.data.Form
 import play.api.i18n.{I18nSupport, MessagesApi}
 import play.api.mvc.{Action, AnyContent, MessagesControllerComponents}
+import services.SaveService
 import uk.gov.hmrc.play.bootstrap.frontend.controller.FrontendBaseController
+import utils.FormUtils.FormOps
 import viewmodels.DisplayMessage.{ListMessage, ListType, Message, ParagraphMessage}
 import viewmodels.implicits._
 import viewmodels.models.{RadioListRowViewModel, RadioListViewModel}
 import views.html.RadioListView
 
 import javax.inject.Inject
+import scala.concurrent.{ExecutionContext, Future}
 
 class PensionSchemeMembersController @Inject()(
   override val messagesApi: MessagesApi,
   navigator: Navigator,
   identifyAndRequireData: IdentifyAndRequireData,
   formProvider: RadioListFormProvider,
+  saveService: SaveService,
   val controllerComponents: MessagesControllerComponents,
   view: RadioListView
-) extends FrontendBaseController
+)(implicit ec: ExecutionContext)
+    extends FrontendBaseController
     with I18nSupport {
 
   private val form = PensionSchemeMembersController.form(formProvider)
 
   def onPageLoad(srn: Srn): Action[AnyContent] = identifyAndRequireData(srn) { implicit request =>
-    Ok(view(form, viewModel(srn, request.schemeDetails.schemeName)))
+    Ok(
+      view(
+        form.fromUserAnswers(PensionSchemeMembersPage(srn, Manual)),
```

Comment:
Since we are saving the selection here I don't think we need to have `Manual` or `Upload` as part of the page case class - we can just pull it from useranswers in the navigator to chose where to go next 

It also makes this more readable as hardcoding Manual doesn't make sense